src/sagemaker_xgboost_container/algorithm_mode/train.py

```diff
@@ -221,9 +221,19 @@ def train_job(train_cfg, train_dmatrix, val_dmatrix, model_dir, checkpoint_dir,
         logging.info("Validation matrix has {} rows".format(val_dmatrix.num_row()))
 
     try:
+        nfold = train_cfg.pop("nfold", None)
+
         bst = xgb.train(train_cfg, train_dmatrix, num_boost_round=num_round, evals=watchlist, feval=configured_feval,
                         early_stopping_rounds=early_stopping_rounds, callbacks=callbacks, xgb_model=xgb_model,
                         verbose_eval=False)
+
+        if nfold is not None:
+            all_dmatrix = train_dmatrix
+            # TODO: XGBoost doesn't support DMatrix concatenation.
+            #if val_dmatrix is not None:
```

Comment:
CodeBuild is failing due to this line:
```
src/sagemaker_xgboost_container/algorithm_mode/train.py:233:13: E265 block comment should start with '# '
975 | ERROR: InvocationError for command /app/.tox/flake8/bin/flake8 setup.py src test (exited with code 1)
976 | ___________________________________ summary ____________________________________
977 | py37-xgboost1.2: commands succeeded
978 | ERROR:   flake8: commands failed
979
```

Reply:
Thanks. I fixed it. More importantly, are you aware of anyway to concatenate DMatrix?